### ISSUE TITLE
Issues downloading pluggable driver from github

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#18075]
Version: Master 2.2 (2/13/19)

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Try to add this cluster driver:
https://github.com/rancher/kontainer-engine-driver-example/releases/download/v0.2.1/kontainer-engine-driver-example-linux

**Result:**
The UI will sit there for a long time in the Downloading state but this usually takes under 20 seconds. After a long time (2 minutes?) it will say Active but that may not actually be true.
If you use docker exec -it <container> bash to get a shell on the rancher server install wget and try to wget the binary it will not succeed stating that there is an issue with github's certificate.

This makes me think the driver is possibly not installing correctly.

Note you can download the binary without issue on Chrome.

**Comment by davidnuzik** [comment]
This was previously working in #15101 a few weeks ago. Now the ui-driver-skel is not working and we think that on the backend downloading the driver might possibly be the cause. We do not have 100% confirmation that there is indeed an issue on the backend it's possible the internet is just slow; we didn't spot anything in logs.

**Comment by cjellick** [comment]
@davidnuzik its unclear to me what happened here. does it do this consistently?

**Comment by davidnuzik** [comment]
@cjellick both @soumyalj and I were running into this the other day when she asked my help to set up ui-driver-skel for it. When we watched the cluster drivers page in the UI it took a rather long time to download the binary for the pluggable driver -- much longer than normal. However we saw nothing in the logs to indicate any actual problem. Perhaps this issue should be more of a question and needs review with a dev because we could not identify the root cause.

It's possible that the issue is just ui-driver-skel but it just didn't seem right that the pluggable driver took such a long time to download it seemed like something was wrong.
